### PR TITLE
Correct typos

### PR DESCRIPTION
**Pull request by gregcotten**
Found a handful of typos when looking through the documentation and figured I'd run `typos-cli` on the codebase.

### CODE REVIEW

File: Sources/GateEngine/System/Rendering/Resources/Text/Backends/TTFFont.swift
Comment (STREGA) [inline Sources/GateEngine/System/Rendering/Resources/Text/Backends/TTFFont.swift:1]: This one was spelled incorrectly on purpose to distinguish the variable from the values being stored.

Let's leave it the way it was.

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/SAMPLER_DESC.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/SAMPLER_DESC.swift:1]: This one was okay. Its an acronym.

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/STATIC_SAMPLER_DESC.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/STATIC_SAMPLER_DESC.swift:1]: Same acronym here too

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX1D_ARRAY_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX1D_ARRAY_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX1D_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX1D_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_ARRAY_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_ARRAY_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_ARRAY_UAV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_ARRAY_UAV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX2D_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX3D_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEX3D_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEXCUBE_ARRAY_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEXCUBE_ARRAY_SRV.swift:1]: acronym again

File: Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEXCUBE_SRV.swift
Comment (STREGA) [inline Sources/GateEngineDependencies/Direct3D12/d3d12_h/Structures/TEXCUBE_SRV.swift:1]: acronym again

File: .typos.toml
Comment (STREGA) [inline .typos.toml:1]: I'd like to keep the repository root clean of unnecessary config files. 

They can be intimidating and make a project feel more complicated to use then it really is.

Can this config be done through arguments? Maybe we could set this up as a package plugin.

**Review by STREGA — CHANGES_REQUESTED** [review]
Most of these look good to me. 

A few files weren't spelling errors though. 
The Windows build error looks like its from those as well, so undoing them should fix that.

**Comment by gregcotten** [comment]
@STREGA ok changes made! Yes, I didn't catch LOD getting autocorrected to LOAD 👀

**Review by STREGA — APPROVED** [review]

**Pull request merged**